README.md

```diff
@@ -1,3 +1,20 @@
+###### (C) Copyright ${year} core.ai (https://core.ai/) 
```

Comment:
GPL clause is only necessary for code files, ie. *.py files. It can be omitted for all other files.

Reply:
What about .yaml files, .toml files, and .cfg files?

Reply:
We can omit those too. It might be problematic if someone uses custom parsers for config files that do not support comments.